### DISCUSSION TITLE
Are there plans for Wayland support outside GNOME?

### DISCUSSION THREAD

**Discussion by jjramsey** [rcaelers/workrave#436] [Q&A]
I asked about this on the ye olde Sourceforge mailing list, but I'm not sure it's used anymore.

I know that there's Wayland support if Mutter is used, but last time I checked, I hadn't seen support for any other Wayland compositor. It looks like the KDE idle protocol (https://wayland.app/protocols/kde-idle, https://gitlab.freedesktop.org/wayland/wayland-protocols/-/merge_requests/29) is supported not just by KDE but by wlroots, too. Not too sure that it's all that mature, but it does seem to be used by swayidle, which in turn is used by an RSI/eye-break timer called SafeEyes.

Are there any plans to support this idle protocol in Workrave, or some other way to get Wayland support outside GNOME?

**Comment by mkesper** [comment]
There's now an issue tracking support: https://github.com/rcaelers/workrave/issues/523
